samtranslator/model/eventsources/push.py

```diff
@@ -471,6 +472,37 @@ def _add_swagger_integration(self, api, function):
                     method=self.Method, path=self.Path))
 
         editor.add_lambda_integration(self.Path, self.Method, uri)
+
+        if self.Auth:
+            method_authorizer = self.Auth.get('Authorizer')
+
+            if method_authorizer:
```

Comment:
Looking at the test coverage, it shows that lines 479-504 (this section) were missed in the tests. Could you make sure that tests are hitting this section?

Reply:
The terminal coverage report isn't clear. If you look at the html report it is a "partial" miss. In this case it's saying that `method_authorizer` was never false.

Reply:
My mistake. Thank you